gasp-node/rollup/runtime/src/lib.rs

```diff
@@ -167,10 +167,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	//   `spec_version`, and `authoring_version` are the same between Wasm and native.
 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
 	//   the compatible custom types.
-	spec_version: 102,
+	spec_version: 104,
 	impl_version: 2,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 1,
+	transaction_version: 2,
```

Comment:
previous release PR was not merged into monorepo
this PR sets previous values and incs the spec_version +1

Reply:
I guess we must merge it, isn't it ?

Reply:
not necessary, changes in mangata-node release PR are part of this PR

Reply:
<img width="602" alt="image" src="https://github.com/user-attachments/assets/d8114f4d-e62d-4491-9b9d-dab515c98584" />